papis/commands/bibtex.py

```diff
@@ -61,8 +61,8 @@
 
     papis bibtex add -q einstein save
 
-Check references quality
-^^^^^^^^^^^^^^^^^^^^^^^^
+Check references' quality
```

Comment:
```suggestion
Check reference quality
```
?

Reply:
Gotta make sure to make the underline shorter too :smile: (note to self)